helm_deploy/values-preprod.yaml

```diff
@@ -13,6 +13,7 @@ generic-service:
   env:
     HMPPS_AUTH_URL: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth
     PRISON_API_URL: https://api-preprod.prison.service.justice.gov.uk
+    PRISONER-OFFENDER-SEARCH_URL: https://prisoner-offender-search-preprod.prison.service.justice.gov.uk
```

Comment:
as above

Reply:
updated to PRISONER_OFFENDER_SEARCH_URL.